cmd/go-runner/plugins/fault_injection.go

```diff
@@ -94,3 +94,7 @@ func (p *FaultInjection) Filter(conf interface{}, w http.ResponseWriter, r pkgHT
 		log.Errorf("failed to write: %s", err)
 	}
 }
+
+func (p *FaultInjection) RespFilter(interface{}, pkgHTTP.Response) {
```

Comment:
Since Go std library doesn't use RespWriter, it would be better to use ResponseFilter as the API name.